src/scenes/Wallet/WalletFund.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { SafeOnRampKit, SafeOnRampProviderType } from '@safe-global/onramp-kit'
+
+export interface WalletFundProps {
+  address: string;
+};
+
+function WalletFund() {
+
+  const [address, setAddress] = useState<string>(localStorage.getItem('safeAddress')||'');
+
+  function handleAddressChange(event: React.ChangeEvent<HTMLInputElement>) {
+    setAddress(event.target.value);
+  }
+
+  const fundWallet = async function() {
+        
```

Comment:
Any particular reason to leave several spaces here and in other places?

Reply:
Removed in https://github.com/5afe/safe-space/pull/8/commits/f055193a106de1b06cc4de83d91c6bec0c29a727